crates/router/src/routes/admin.rs

```diff
@@ -644,7 +516,7 @@ pub async fn connector_update(
             )
         },
         auth::auth_type(
-            &auth::AdminApiAuthWithMerchantId::default(),
+            &auth::AdminApiAuthWithMerchantIdFromHeader,
```

Comment:
Isn't this going to affect existing Postman or Cypress tests? Shouldn't we be updating the v2 merchant connector update route instead?

Reply:
The `default()` checks the merchant id from header. I have just renamed this